scripts/pre-release

```diff
@@ -2,23 +2,63 @@
 # Announce the script so we know we're in the right place.
 echo "Welcome to the release prep script!"
 
-# Get the release version.
-echo "Enter the release version: "
-read VERSION
+# Check if this is a pre-release or a regular release.
```

Comment:
I'm confused a bit the name of the script and the release.

I think there are two types of release: pre-release and regular release.
I also think there are two types of scripts: pre-release and post-release

If that's the case, we're good.

Reply:
I believe there are ways that releases can be created with the `gh` commandline tool, but to do that, we'd also need to automate the process of pulling in the PR names to build release notes. Altis has some scripts for this, but I don't know if they're public, so for now, the actual _creating a release_ part is manual.